modules/features/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ ~ Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~      http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+-->
+        
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    
+    <parent>
+      <groupId>org.apache.synapse</groupId>
+      <artifactId>Apache-Synapse</artifactId>
+      <version>2.1.7-wso2v118-SNAPSHOT</version>
+      <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>synapse-features</artifactId>
+    <packaging>pom</packaging>
+    <name>WSO2 Carbon - Synapse Feature Aggregator Module</name>
+    <url>http://wso2.org</url>
+
+    <modules>
+        <module>org.apache.synapse.wso2.feature</module>
+	    <module>org.apache.synapse.transport.fix.feature</module>
```

Comment:
please do formatting

Reply:
Done in https://github.com/wso2/wso2-synapse/pull/1391/commits/d9ab8a682e834be7903af1bfd95939f6014c9597